LCM/scripts/RemoveModule.py

```diff
@@ -50,6 +50,37 @@ def RemoveMirroredDirectory(moduleDir, omiDir):
 
 print("Removing module " + moduleName)
 
+# special section for nxOMSAutomationWorker module
+# the Linux Hybrid worker needs to be killed before the module is removed
+# also a good idea to remove the state.conf file
+# we preserve the worker.conf file in case the user wants to preserve his settings
+if moduleName == "nxOMSAutomationWorker":
+    state_conf_location = "/var/opt/microsoft/omsagent/state/automationworker/state.conf"
```

Comment:
This location isn't necessarily correct for multi-homed agents. I realize DSC isn't assumed to be fully compatible with multi-homing yet, but maybe search through /var/opt/microsoft/omsagent/<regex for wksp id>/state directories as well for state.conf.